File: python/hello-world/hello.py
# I could have simply done:
# print("Hello 世界!")
# But that seemed too simple, so I went with template literals

greeting = "Hello"
name = "世界"

print('String example 1:')
print(f'{greeting} {name}!')
# The code above is the newest way of writing template literals
# However, we could also use the old way of writing template literals
# The code below demonstrates this
print('String example 2:')
print('%s %s!' % (greeting, name))

File: typescript/hello-world/hello.ts
// This first example is a very simple and easy
let greeting:string = 'Hello';
let world:string = '世界';

console.log('String 1:');
console.log(`${greeting} ${world}!`);


// This second example demonstrates the same thing,
// but with an interface and template literals
interface WorldGreeting {
  greeting: string,
  name: string
};

const newGreeting: WorldGreeting = {
  greeting: 'Hello',
  name: '世界'
};

console.log('String 2:');
console.log(`${newGreeting.greeting} ${newGreeting.name}!`);